CONTRIBUTING.md

```diff
@@ -30,9 +30,8 @@ Standards
 ansible-lint is flake8 compliant with `max-line-length` set to 100
 (see [setup.cfg](setup.cfg)).
 
-ansible-lint works with both Ansible 1.9 and Ansible 2.0 onwards. This will be
-the case for the foreseeable future, so please ensure all contributions work
-with both.
+ansible-lint works only with [supported Ansible versions](https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#release-status) at the time it was
```

Comment:
Let's make this line shorter.
```suggestion
ansible-lint works only with [supported Ansible versions](
https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#release-status
) at the time it was
```
Alternatively, we could use a separate detached anchor
```
[supported Ansible versions]:
https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#release-status
```